lib/source_readers/flat.rb

```diff
@@ -20,13 +20,14 @@ def self.resolve(target)
       new(target, files)
     end
 
-    attr_reader :metadata, :tests, :libraries
+    attr_reader :metadata, :tests, :libraries, :files
```

Comment:
👍 yeah had a similar feeling, just not a good name. i'll go with `data_files`, can't thing of anything else and it looks good